libs/shared/src/consts/providers/provider-credentials.ts

```diff
@@ -7,11 +7,13 @@ const mailConfigBase: IConfigCredentials[] = [
     displayName: 'From email address',
     description: 'Use the authenticated email address from the delivery provider you will send emails from.',
     type: 'string',
+    required: true,
```

Comment:
The `required: true` is kinda repetitive since more than 90% of the fields are **required** but I thought this would make it easier since we don't have handle `undefined` when consuming the `IConfigCredentials` interface in different places.
Lemme know what you think @scopsy

Reply:
I think it's perfectly fine and makes a clear API 🥳